core/domain/exp_domain.py

```diff
@@ -2187,6 +2187,28 @@ def _convert_states_v28_dict_to_v29_dict(cls, states_dict):
             state_dict['solicit_answer_details'] = False
         return states_dict
 
+    @classmethod
+    def _convert_states_v29_dict_to_v30_dict(cls, states_dict):
```

Comment:
Are these methods tested?

Reply:
Yes. Please see SchemaMigrationUnitTests in exp_domain_test.py.